### PR TITLE
Feature/cax portal@redux store alternative

### PR DESCRIPTION
**Pull request by vsamarora**
CPLP 374 Front end State Management Framework: Switch Mobx to Redux changes

### CODE REVIEW

**Comment by fatihayyildiz** [comment]
Thanks Samidha, I am checking out the changes

File: portal/code/company-portal/src/components/cax-header.tsx [hunk 1]
```diff
@@ -14,50 +14,41 @@
...
-        const token = UserService.getToken();
-        console.log(token);
-    }
 
-    private logoutClick() {
-        const token = UserService.getCachedToken();
-        console.log(token);
-        UserService.logOut();
+    const logoutClick = () => {
+        // const token = UserService.getCachedToken();
```
Comment (oyo) [inline portal/code/company-portal/src/components/cax-header.tsx]: maybe remove those comments - actually it becomes a useless wrapper function. see my comment below

File: portal/code/company-portal/src/components/cax-header.tsx [hunk 2]
```diff
@@ -71,16 +62,16 @@ class Header extends React.Component<IProp> {
...
                         <div className='profile-link user'>
                             <input id="myid" type="checkbox" />
                             <label htmlFor="myid" className='user-icon'></label>
-                            <span className="tooltiptext"> <div> {this.username}</div>
+                            <span className="tooltiptext"> <div> {username}</div>
                                 < div > {UserService.getDomain()}</div>
-                                <div>({this.userRoles.join(", ")})</div>
-                                <div className='logout' onClick={() => this.logoutClick()}>Logout</div>
+                                <div>({userRoles.join(", ")})</div>
+                                <div className='logout' onClick={() => logoutClick()}>Logout</div>
```
Comment (oyo) [inline portal/code/company-portal/src/components/cax-header.tsx]:
```suggestion
                                <div className='logout' onClick={() => UserService.doLogout()}>Logout</div>
```

Comment (oyo) [reply portal/code/company-portal/src/components/cax-header.tsx]: although "Logout" is probably understood by everyone - shouldn't this string be i18n

**Review by oyo — COMMENTED** [review]
just some small remarks from my side. nothing critical though

File: portal/code/company-portal/src/components/cax-header.tsx (hunk 2 above)
Comment (vsamarora) [reply portal/code/company-portal/src/components/cax-header.tsx]: updated the changes

**Review by fatihayyildiz — APPROVED** [review]
Looks alright at my side

**Pull request merged**